usermods/audioreactive/audio_reactive.h

```diff
@@ -579,7 +666,7 @@ class AudioReactive : public Usermod {
       Serial.print("micReal:");     Serial.print(micDataReal); Serial.print("\t");
       Serial.print("volumeSmth:");  Serial.print(volumeSmth);  Serial.print("\t");
       //Serial.print("volumeRaw:");   Serial.print(volumeRaw);   Serial.print("\t");
-      //Serial.print("DC_Level:");    Serial.print(micLev);      Serial.print("\t");
+      Serial.print("DC_Level:");    Serial.print(micLev);      Serial.print("\t");
```

Comment:
Do we really need Serial or would we be better of converting to DEBUG_PRIINT?

Reply:
Good point. The intention  is that this is used with Arduino Serial Plotter, so works best if no other debug output comes on serial. But surely will also work over network with DEBUGOUT. Maybe best if I create a macro like PLOT_PRINT that is defined like DEBUG_PRINT.